osu.Game/Overlays/LoginOverlay.cs

```diff
@@ -79,5 +81,29 @@ protected override void PopOut()
             settingsSection.Bounding = false;
             this.FadeOut(transition_time);
         }
+
+        protected override bool OnDragStart(InputState state)
+        {
+            if (!state.Mouse.IsPressed(MouseButton.Left)) return false;

```

Comment:
this stuff shouldn't be required. we can't just add this everywhere a context menu is being used.